config/post-install/storage-version-migration.yaml

```diff
@@ -15,7 +15,7 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: storage-version-migration
+  generateName: storage-version-migration-
```

Comment:
is there some "magic" needed for this? instead of just name?

Reply:
https://kubernetes.io/docs/reference/using-api/api-concepts/#generated-values

> if generateName is set, name will have a unique random name

The only downside is you can't `kubectl apply` you'll need to do a `kubectl create`